README.md

```diff
@@ -17,126 +19,135 @@ Finally, I'm sure everyone who reads this list has one thing they want to add. P
+* [Hardy](https://github.com/thingsinjars/Hardy) - Selenium-driven, cucumber-powered CSS testing.
+* [TestCafe](https://github.com/DevExpress/testcafe) - Automated browser testing for the modern web development stack.
+* [Needle](https://github.com/python-needle/needle) - Needle is a tool for testing visuals with Selenium and nose (Python).
+* [dpxdt [Depicted]](https://github.com/bslatkin/dpxdt) - End-to-end testing with Python.
+* [gatling](https://github.com/gabrielrotbart/gatling) - Integrated visual RSpec matcher which makes real visual testing easy (Ruby).
+* [grunt-photobox](https://github.com/stefanjudis/grunt-photobox) - Plugin to prevent your project of broken layout via screenshot photo sessions of your site.
+* [vrtest](https://github.com/nathanmarks/vrtest) - JavaScript library for running visual regression tests on your components cross browser via selenium.
+* [Happo](https://github.com/Galooshi/happo) - Visual diffing in CI for user interfaces.
+* [reg-cli](https://github.com/bokuweb/reg-cli) - Visual regression test tool which output easy-to-read single file html report.
+* [Nightmare](https://github.com/segmentio/nightmare) - High-level browser automation library based on Electron.
+* [Puppeteer](https://github.com/GoogleChrome/puppeteer) - Headless Google Chrome Node API.
+* [Visual Review](https://github.com/xebia/VisualReview) - A human-friendly tool for testing and reviewing visual regressions.
+* [reg-suit](https://github.com/reg-viz/reg-suit) - Visual regression testing suite which compares images, stores snapshots, and notifies the difference to your GitHub repo.
+* [Chimp](https://github.com/xolvio/chimp) - Develop acceptance tests & end-to-end tests with realtime feedback.
+* [Differencify](https://github.com/NimaSoroush/differencify) - A library for visual regression testing using [Puppeteer](https://github.com/GoogleChrome/puppeteer).
+* [ResembleJS](https://github.com/Huddle/Resemble.js) - Analyse and compare images with Javascript and HTML5.
+* [Muppeteer](https://github.com/HuddleEng/Muppeteer) - Visual regression testing framework for Chrome using [Mocha](https://mochajs.org/) and [Puppeteer](https://github.com/GoogleChrome/puppeteer).
+* [ember-visual-test](https://github.com/Cropster/ember-visual-test) - Simple visual regression testing for [Ember](https://emberjs.com/).
+* [AET](https://github.com/Cognifide/aet) - Scalable testing tool providing visual regression testing, accessibility and performance validation, markup analysis and more.
+* [Wendigo](https://github.com/angrykoala/wendigo) - Test-oriented browser automation library based on Puppeteer. 
+* [Loki](https://github.com/oblador/loki) - Visual regression testing for Storybook using Chrome in docker et al. 
+* [Zombie.js](http://zombie.js.org/) - Insanely fast, headless full-stack testing using Node.js.
+* [CodeceptJS](https://github.com/codeception/codeceptjs/) - Modern Era Acceptance Testing Framework for NodeJS.
```

Comment:
:+1: Nice! Normally we have one Pull-Request per addition, so we can discuss the changes. You should defininetly seperate "Style"-changes from "content"-contributions.